openlibrary/catalog/marc/tests/test_marc_html.py

```diff
@@ -1,46 +1,15 @@
+import os
```

Comment:
Would be cooler to see `from pathlib import Path` but that is a developer choice.  ;-)